public/components/visualizations/charts/pie/pie.tsx

```diff
@@ -72,18 +71,15 @@ export const Pie = ({ visualizations, layout, config }: any) => {
     xaxes = dimensions;
   }
 
-  if (isEmpty(xaxes) || isEmpty(series)) {
-    return <EmptyPlaceholder icon={icontype} />;
-  }
-
   const invertHex = (hex: string) =>
-    (Number(`0x1${hex}`) ^ HEX_CONTRAST_COLOR).toString(16).substr(1).toUpperCase();
+    (Number(`0x1${hex}`) ^ HEX_CONTRAST_COLOR).toString(16).substr(1).toUpperCase(); // eslint-disable-line no-bitwise
```

Comment:
Not sure if there's quick resolution here, we may need to change the implementation here to pass this linting.